packages/logs/src/domain/loggerSession.ts

```diff
@@ -12,14 +12,7 @@ export enum LoggerTrackingType {
   TRACKED = '1',
 }
 
-export function startLoggerSession(configuration: Configuration, areCookieAuthorized: boolean): LoggerSession {
-  if (!areCookieAuthorized) {
```

Comment:
We probably want to keep this

Reply:
The check is done in [startLogs](https://github.com/DataDog/browser-sdk/blob/c0356a2b8fb852dba1d40689fbd09217011bf650/packages/logs/src/boot/startLogs.ts#L40)

Reply:
Ah, I missed it. Nice!